### ISSUE TITLE
Terminal pins randomly at times

### ISSUE THREAD

**Issue by lucian303** [binaryage/totalterminal#60]
This issue goes back at least 2 years. After awhile of usage the terminal pins itself for no reason and I can't cmd tab to other apps. Well to clarify, I can, and it'll switch but the window stays. I just realized this is happening because of pinning as I can generally (but not all the time) hit unpin and have it work. 

It's been a horrific bug. When I can't unpin it, I reassign a pin shortcut and reassign back. Then it works. However, having this not happen ever unless the key combo is pressed or adding an option to disable pinning altogether. 

I'm not sure what causes it, but I have been able to replicate it on:
Snow Leopard, Lion, Mountain Lion on a late 2007 MBPro.
Lion and MountainLion on 2011 MBPro 13, and 2 17"
Mountain Lion on two 15" Retinas
Mountain Lion on 2012 Air

**Comment by RobertDeRose** [comment]
I am having the same issue.  I was able to use the workaround stated in the above issue of un-assigning the pin shortcut and reassigning it, then pinning and unpinning Visor.  Works as expected now.

Would be nice to have feature on/off switch in preferences.

**Comment by lucian303** [comment]
Indeed. I've found lately that just hitting the pin/unpin shortcut once (to unpin) also works for me and then TT starts behaving normally again. I have no idea what is causing it, but it is certainly not hitting the shortcut or pinning it intentionally in any way.
